src/App.jsx

```diff
@@ -283,6 +303,17 @@ class App extends Component {
     this.setState({ showReadyLight: false });
   }
 
+  bypass2FA () {
+    const queryString = this.props.location.search;
+    const query = new URLSearchParams(queryString);
+    const cid = query.get('cid');
+    const voterDeviceId = VoterStore.voterDeviceId();
+    if (cid && cid !== voterDeviceId) {
+      VoterStore.setVoterDeviceIdCookie(cid);
```

Comment:
I think we want to call VoterSessionActions.setVoterDeviceIdCookie instead of VoterStore.setVoterDeviceIdCookie.